samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/CallingBot.cs

```diff
@@ -142,6 +151,25 @@ private async Task NotificationProcessor_OnNotificationReceivedAsync(Notificatio
 
                 var recordingLocation = await teamsRecordingService.DownloadRecording(recording.RecordingLocation, recording.RecordingAccessToken);
 
+                try
+                {
+                    var callDetails = callService.Get(callId);
+                    var result = await speechService.ConvertWavToText(recordingLocation);
+
+                    if (result.Reason == ResultReason.RecognizedSpeech)
+                    {
+                        var threadId = (await callDetails)?.ChatInfo?.ThreadId;
+                        if (threadId != null)
+                        {
+                            await botService.SendToConversation($"You said: {result.Text}", threadId);
+                        }
+                    }
```

Comment:
For completeness let's print something in the other cases too (other ResultReason values)